app/javascript/tests/analytics/EnrollerAnalytics.test.js

```diff
@@ -7,20 +7,33 @@ import { mockEnrollerStatistics1 } from '../mocks/mockEnrollerStatistics';
 
 describe('EnrollerAnalytics', () => {
   it('Properly Renders Both EnrollerStatistics Cards correctly', () => {
-    const eaWrapper = shallow(<EnrollerAnalytics user={mockUser1} stats={mockEnrollerStatistics1}/>);
-    expect(eaWrapper.find('Fragment').find('Row').find('Col').at(0).containsMatchingElement(<EnrollerStatistics />)).toBeTruthy();
+    const eaWrapper = shallow(<EnrollerAnalytics user={mockUser1} stats={mockEnrollerStatistics1} />);
+    expect(
```

Comment:
I wonder if changing this line to `expect(wrapper.find(EnrollerStatistics).exists()).toBeTruthy();` would fix this? we don't necessarily need to test that it's in that specific location

Reply:
Sure! Im not against that. Fixed in latest commit